udata_front/theme/gouvfr/assets/js/locales/en.json

```diff
@@ -483,5 +483,9 @@
   "Member since": "Member since",
   "Last connection": "Last connection",
   "No connection": "No connection",
-  "You are seeing a specific community resource of this dataset": "You are seeing a specific community resource of this dataset"
+  "You are seeing a specific community resource of this dataset": "You are seeing a specific community resource of this dataset",
+  "Access terms": "Access terms",
+  "Open APIs to everyone": "Open APIs to everyone",
+  "Restricted access APIs": "Restricted access APIs",
+  "{count} API": "{count} API"
```

Comment:
Do we want to show plurals for this one ?

Reply:
There is no plural in the Figma but we could add it…

Reply:
Let's keep it without plural then